web/src/pages/Route/List.tsx

```diff
@@ -63,6 +104,84 @@ const Page: React.FC = () => {
     });
   };
 
+  const handleExport = (exportFileType: number) => {
+    exportRoutes(selectedRowKeys.join(',')).then((resp) => {
+      let exportFile: string;
+      let exportFileName = `APISIX_routes_${moment().format('YYYYMMDDHHmmss')}`;
+
+      switch (exportFileType) {
+        case ExportFileType.YAML:
+          exportFile = yaml.dump(resp.data);
+          exportFileName = `${exportFileName}.yaml`;
+          break;
+        case ExportFileType.JSON:
+        default:
+          exportFile = js_beautify(JSON.stringify(resp.data), {
+            indent_size: 2,
+          });
+          exportFileName = `${exportFileName}.json`;
+          break;
+      }
+
+      const blob = new Blob([exportFile], {
+        type: EXPORT_FILE_MIME_TYPE_SUPPORTED[exportFileType],
+      });
+      const aLink = document.createElement('a');
```

Comment:
Please take a look at `file-saver` if it's helpful

https://github.com/eligrey/FileSaver.js

Reply:
> Please take a look at `file-saver` if it's helpful
> 
> https://github.com/eligrey/FileSaver.js

so cool~! it would be better than native approach, let me updated ASAP, thanks a lot  @juzhiyuan

Reply:
fixed